### ISSUE TITLE
Calling the Down method on MigrationRunner for in-process does not remove the version from VersionInfo table

### ISSUE THREAD

**Issue by gandarez** [fluentmigrator/fluentmigrator#1127]
When calling the `down(IMigration migration)` on MigrationRunner for in-process does not remove the version from VersionInfo table which is actually weird because I run the down for this specific migration and for that reason turns the control table inconsistent.

**Comment by jzabroski** [comment]
I don't think it should. The VersionInfo table is an immutable record of what happened.  Probably the thing missing is a column to indicate up vs. down.

**Comment by gandarez** [comment]
In my opinion that is a misunderstaing what the real purpose for rollback versus down. Is that documented?

**Comment by jzabroski** [comment]
The VersionInfo table is an immutable record of what happened, aka audit trail.  The thing missing is a column indicating up vs. down.  I am not sure I understand your comment about rollback versus down.  Rollback is an abstraction boundary for collecting down scripts.  The Runner doesn't log those higher-level concepts by default.  You could add a RunnerCommandHistory table if you'd like to your In Process Runner.
